synfig-studio/src/gui/trees/childrentreestore.cpp

```diff
@@ -79,11 +79,13 @@ ChildrenTreeStore::ChildrenTreeStore(etl::loose_handle<synfigapp::CanvasInterfac
 	canvas_row[model.label]=_("Canvases");
 	canvas_row[model.is_canvas] = false;
 	canvas_row[model.is_value_node] = false;
+	canvas_row[model.canvas] = canvas_interface_->get_canvas();
 
```

Comment:
Thanks ^_^ ,

Yes this is similar to what I was thinking of as another solution. Since we really don't need to edit anything here we can just disable the cell renderers for this row.

however using something like:
`canvas_row[model.is_editable] = false;`
For some reason is still not making the cell not editable... i.e. when the `start_editing_vfunc` of the cell renderer gets called `property_editable()` returns true, although it seems all this model stuff is initialized as should be in the method responsible for making the cell renderer (`CanvasTreeStore::add_cell_renderer_value`).

Reply:
@rodolforg, done check comment.